frontend/public/components/_masthead.scss

```diff
@@ -36,13 +54,38 @@
     display: flex;
     height: 100%;
   }
+  &__modal {
```

Comment:
I don't think we should need most of these styles as long as we're using the patternfly-react components to render the modal (`AboutModal.Versions`, etc)